### PR TITLE
Fix qCC_db typos

### PR DESCRIPTION
**Pull request by tmontaigu**
Fix a bunch of typos in comments/documentation.

And use \warning for better doxygen output.

There are some typos in methods names, should we also correct them ?

### CODE REVIEW

**Comment by dgirardeau** [comment]
Yes, don't hesitate!

**Review by dgirardeau — APPROVED** [review]

**Review by dgirardeau — APPROVED** [review]

**Review by dgirardeau — CHANGES_REQUESTED** [review]
It seems something is broken?

**Comment by tmontaigu** [comment]
Yes, on my local build I do not build a lot of plugins, so some renamed function were missed by the refactoring tool

File: libs/CCPluginAPI/include/ccOverlayDialog.h [hunk 1]
```diff
@@ -60,7 +60,7 @@ class CCPLUGIN_LIB_API ccOverlayDialog : public QDialog
 	//! Adds a keyboard shortcut (single key) that will be overridden from the associated window
 	/** When an overridden key is pressed, the shortcutTriggered(int) signal is emitted.
 	**/
-	void addOverridenShortcut(Qt::Key key);
+	void addoverriddenShortcut(Qt::Key key);
```
Comment (dgirardeau) [inline libs/CCPluginAPI/include/ccOverlayDialog.h:63]: Why removing the capital 'O'?

File: libs/qCC_db/include/ccDrawableObject.h [hunk 2]
```diff
@@ -155,7 +155,8 @@ class QCC_DB_LIB_API ccDrawableObject
 	virtual void setTempColor(const ccColor::Rgb& col, bool autoActivate = true);
 
 	//! Set temporary color activation state
-	inline virtual void enableTempColor(bool state) { m_colorIsOverriden = state; }
+	inline virtual void enableTempColor(bool state) {
```
Comment (dgirardeau) [inline libs/qCC_db/include/ccDrawableObject.h:158]: Not sure about the way to format this

File: libs/qCC_db/src/ccDrawableObject.cpp [hunk 3]
```diff
@@ -43,7 +43,8 @@ ccDrawableObject::ccDrawableObject(const ccDrawableObject& object)
 	, m_normalsDisplayed(object.m_normalsDisplayed)
 	, m_sfDisplayed(object.m_sfDisplayed)
 	, m_tempColor(object.m_tempColor)
-	, m_colorIsOverriden(object.m_colorIsOverriden)
+	,
```
Comment (dgirardeau) [inline libs/qCC_db/src/ccDrawableObject.cpp:46]: should be on one line

File: libs/qCC_db/src/ccHObject.cpp [hunk 4]
```diff
@@ -1059,19 +1059,21 @@ bool ccHObject::fromFile(QFile& in, short dataVersion, int flags, LoadedIDMap& o
...
-			//we need to load the custom object as plain ccCustomHobject
+			//we need to load the custom object as plain ccCustomHObject
 			child->fromFileNoChildren(in, dataVersion, flags, oldToNewIDMap);
 			//go back to original position
 			in.seek(originalFilePos);
 			//get custom object name and plugin name
 			QString childName = child->getName();
-			QString classId = child->getMetaData(ccCustomHObject::DefautMetaDataClassName()).toString();
-			QString pluginId = child->getMetaData(ccCustomHObject::DefautMetaDataPluginName()).toString();
+			QString classId = child->getMetaData(
```
Comment (dgirardeau) [inline libs/qCC_db/src/ccHObject.cpp:1073]: should be on one line (both)

File: libs/qCC_glWindow/include/ccGLWindow.h [hunk 5]
```diff
@@ -471,7 +471,7 @@ class CCGLWINDOW_LIB_API ccGLWindow : public ccGLWindowParent, public ccGenericG
 	void setDisplayParameters(const ccGui::ParamStruct& params, bool thisWindowOnly = false);
 
 	//! Whether display parameters are overidden for this window
-	bool hasOverridenDisplayParameters() const { return m_overridenDisplayParametersEnabled; }
+	bool hasoverriddenDisplayParameters() const { return m_overriddenDisplayParametersEnabled; }
```
Comment (dgirardeau) [inline libs/qCC_glWindow/include/ccGLWindow.h:474]: capital O

File: libs/qCC_glWindow/src/ccGLWindow.cpp [hunk 6]
```diff
@@ -977,7 +977,7 @@ bool ccGLWindow::initialize()
 #endif
 
 		//apply (potentially) updated parameters;
-		setDisplayParameters(params, hasOverridenDisplayParameters());
+		setDisplayParameters(params, hasoverriddenDisplayParameters());
```
Comment (dgirardeau) [inline libs/qCC_glWindow/src/ccGLWindow.cpp:980]: capital O

File: libs/qCC_glWindow/src/ccGLWindow.cpp [hunk 7]
```diff
@@ -1002,7 +1002,7 @@ bool ccGLWindow::initialize()
 				{
 					m_customRenderingShader = renderingShader;
 				}
-				setDisplayParameters(params,hasOverridenDisplayParameters());
+				setDisplayParameters(params,hasoverriddenDisplayParameters());
```
Comment (dgirardeau) [inline libs/qCC_glWindow/src/ccGLWindow.cpp:1005]: same

**Review by dgirardeau — CHANGES_REQUESTED** [review]

File: libs/CCPluginAPI/include/ccOverlayDialog.h (hunk 1 above)
Comment (tmontaigu) [reply libs/CCPluginAPI/include/ccOverlayDialog.h]: It was a mistake

File: qCC/extern/QCustomPlot/qcustomplot.cpp [hunk 8]
```diff
@@ -287,7 +287,7 @@ QCPPainter::QCPPainter() :
   mModes(pmDefault),
   mIsAntialiasing(false)
 {
-  // don't setRenderHint(QPainter::NonCosmeticDefautPen) here, because painter isn't active yet and
+  // don't setRenderHint(QPainter::NonCosmeticdefaultPen) here, because painter isn't active yet and
```
Comment (dgirardeau) [inline qCC/extern/QCustomPlot/qcustomplot.cpp:290]: Shouldn't it be NonCosmeticDefaultPen? I guess it doesn't compile?

**Review by dgirardeau — CHANGES_REQUESTED** [review]

File: qCC/extern/QCustomPlot/qcustomplot.cpp (hunk 8 above)
Comment (dgirardeau) [reply qCC/extern/QCustomPlot/qcustomplot.cpp:290]: Ah sorry, it's a comment :D

**Review by dgirardeau — APPROVED** [review]

**Comment by dgirardeau** [comment]
Once again, thanks for your work!

**Comment by tmontaigu** [comment]
Thanks for the reviews

**Pull request merged**